testgrid/util/gcs/BUILD.bazel

```diff
@@ -4,7 +4,7 @@ go_library(
     name = "go_default_library",
     srcs = ["gcs.go"],
     importpath = "k8s.io/test-infra/testgrid/util/gcs",
-    visibility = ["//testgrid:__subpackages__"],
+    visibility = ["//visibility:public"],
```

Comment:
Do we really want imports from `prow/` to go to `testgrid/`? What happens when we break out `prow/` from this repo? Do we want a shared (separate) GCS utility package?

Reply:
Feel free to toss in a `TODO(fejta): refactor this into somewhere common`

Will you switch the visibility to:
```
visibility = [
  "//testgrid:__subpackages__",
  "//prow/gcsupload:__subpackages__",
]
```